fluentd/http/values.yaml

```diff
@@ -3,7 +3,7 @@ fluentd:
 
   image:
     repository: coralogixrepo/coralogix-fluentd-multiarch
-    tag: v0.0.5
+    tag: v0.0.6
```

Comment:
the new image [0.0.6] includes only a prometheus plugin  in a newer version, but it seems like Fluentd doesnt let us override the previous version, meaning this tag doesn't actually change anything currently so no need to update it here. 

If you want to change it, then we need to update he CHANGELOG